packages/docusaurus-mdx-loader/src/remark/transformImage/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -14,27 +14,27 @@ exports[`transformImage plugin pathname protocol 1`] = `
 exports[`transformImage plugin transform md images to <img /> 1`] = `
 "![img](https://example.com/img.png)
 
-<img src={require(\\"![CWD]/node_modules/url-loader/dist/cjs.js?limit=10000&name=assets/images/[name]-[hash].[ext]&fallback=[CWD]/node_modules/file-loader/dist/cjs.js!./static/img.png\\").default} />
+<img loading=\\"lazy\\" src={require(\\"![CWD]/node_modules/url-loader/dist/cjs.js?limit=10000&name=assets/images/[name]-[hash].[ext]&fallback=[CWD]/node_modules/file-loader/dist/cjs.js!./static/img.png\\").default} />
```

Comment:
not really related but can you replace the fixtures by real images? (in static folders)

We should have width/height attributes in this snapshot, but instead of have test errors because files are not real images 😅 


```
  console.error
    [ERROR] The image at /Users/sebastienlorber/Desktop/projects/docusaurus/packages/docusaurus-mdx-loader/src/remark/transformImage/__tests__/__fixtures__/static/img.png can't be read correctly. Please ensure it's a valid image.
    Empty file
```

Reply:
I've replaced the two fake images with two images from elsewhere in the project.  As a consequence you can now see width and height being set.